index.html

```diff
@@ -22,6 +21,16 @@
       <div4></div4>
       <div5>
         <time></time>
+        <!-- don't set the display style of div10 to none here,
+      instead set the display style of the lds-ring class to none (in the .js)
+          <div10 style="display: none;">
+          -->
+        <div class="lds-ring">
+          <div></div>
+          <div></div>
+          <div></div>
+          <div></div>
+        </div>
```

Comment:
I've moved the spinner here instead of at the end of the html code, so that it can show up in the middle of the screen.